web/src/app/stats/components/stats-chart/stats-chart.component.scss

```
@@ -21,6 +21,16 @@
   }
 }
 
+.chart-mode-toggle {
+  margin-left: auto;
+  align-self: center;
+
+  ::ng-deep .mat-button-toggle-label-content {
```

Stylelint (17.6.0)

[error] Unexpected unknown pseudo-element selector "::ng-deep" (selector-pseudo-element-no-unknown)

(selector-pseudo-element-no-unknown)